flexmeasures/app.py

```diff
@@ -105,6 +105,11 @@ def create(  # noqa C901
 
     app.reporters = get_classes_module("flexmeasures.data.models.reporting", Reporter)
 
+    # Register Schedulers
+    from flexmeasures.data.models.planning import Scheduler
+
+    app.schedulers = get_classes_module("flexmeasures.data.models.planning", Scheduler)
```

Comment:
The registration of `Scheduler`s is handled in a slightly different way. Only with this, we would miss the `Scheduler`s registered in plugins. Plugins' `Reporter` are discovered in this [line](https://github.com/FlexMeasures/flexmeasures/blob/e4c77cc90187ea9e64b531287becdb74d55e8be1/flexmeasures/utils/plugin_utils.py#L108).

Reply:
I guess, I would need some help.

Reply:
Do I need to register schedulers in plugins too?

Reply:
I added https://github.com/FlexMeasures/flexmeasures/pull/708/commits/7d9f9cc460f5b12d4f71f91777dfe1f600741653 to register schedulers defined in plugins, too. While testing I found that classes defined in `__init__.py` package files weren't picked up, so I added https://github.com/FlexMeasures/flexmeasures/pull/708/commits/83d412b5d08572ab98836354543b54f0d95f3124. @victorgarcia98. can you review these commits?

Reply:
Nice find! I had tested it only on a lower level `__init__.py`.